### PR TITLE
Feature/service entries

### PR DESCRIPTION
**Pull request by MichaelHabermas**
# Description

Summary of your changes
updated recipient and service entry end points

## What work was done?

Details of your changes

## Why was this work done?

Why?

## Type of change

- New feature (non-breaking change which adds functionality)?
- Bug Fix?

## Change Status

- Completed, ready to review and merge?
- In progress?
- etc.

## Checklist

- [ ] Endpoint tested with Postman, or Unit test.
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] There are no merge conflicts

### CODE REVIEW

File: api/recipient/recipientRouter.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 const express = require('express');
-// const DB = require('../utils/db-helper');
```
Comment (santorap) [inline api/recipient/recipientRouter.js:2]: Thanks for removing this one.

File: api/serviceEntries/serviceEntriesModel.js [hunk 2]
```diff
@@ -1,107 +1,99 @@
...
-
-  //     return await findById(newServiceEntriesId);
-  //   } catch (err) {
-  //     throw new Error(err);
-  //   }
 };
 
 // Can only update service_entries data in table on front end
 // Can't update "extra" data that's tacked onto the responses
 // Can't edit: first_name, last_name, service_type_name, status_name _
```
Comment (santorap) [inline api/serviceEntries/serviceEntriesModel.js:89]: I think these comments apply to the previous DB, but I don't see any reason why these fields can't be updated now that we've simplified the structure?

File: api/serviceEntries/serviceEntriesModel.js [hunk 3]
```diff
@@ -1,107 +1,99 @@
...
 const update = (id, object) => {
   return knex('service_entries')
-    .where({ id: id })
-    .first()
+    .where('service_entry_id', id)
     .update(object)
```
Comment (santorap) [inline api/serviceEntries/serviceEntriesModel.js:93]: I assume as long as the object has the correct syntax, you don't need to understand the specific fields/tables getting the update?

**Review by santorap — APPROVED** [review]
Updates look good.

I have a couple of questions about some potentially outdated comments in the code and double-checking that updates to fields not in the main service_entries table can still happen as expected if shown on the front end.

**Review by ahmedseragcodes — APPROVED** [review]
Changes make sense, necessary following database schema redesign

**Pull request merged**